layouts/shortcodes/blurb/wolfi.html

```diff
@@ -0,0 +1 @@
+Wolfi is a community Linux undistro created specifically for containers. This brings distroless to a new level, including additional features targeted at securing the software supply chain of your application environment: comprehensive SBOMs, signatures, daily updates, and timely CVE fixes.
```

Comment:
Worth a link to the [Github org](https://github.com/wolfi-dev/)? I dunno, just a thought!